app/src/main/res/layout/option_activity.xml

```diff
@@ -1,31 +1,43 @@
-<androidx.drawerlayout.widget.DrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:app="http://schemas.android.com/apk/res-auto"
-  xmlns:tools="http://schemas.android.com/tools"
-  android:id="@+id/options_activity_drawer_layout"
-  android:layout_width="match_parent"
-  android:layout_height="match_parent">
+  xmlns:profile="http://schemas.android.com/tools">
 
-  <LinearLayout
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:orientation="vertical">
+  <data>
+    <variable
+      name="viewModel"
+      type="org.oppia.app.options.OptionsActivity" />
+  </data>
 
-    <include
-      android:id="@+id/options_activity_toolbar"
-      layout="@layout/toolbar" />
+  <androidx.drawerlayout.widget.DrawerLayout
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/options_activity_drawer_layout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
 
-    <FrameLayout
-      android:id="@+id/options_fragment_placeholder"
+    <LinearLayout
       android:layout_width="match_parent"
-      android:layout_height="match_parent" />
-  </LinearLayout>
+      android:layout_height="match_parent"
+      android:orientation="vertical">
 
-  <fragment
-    android:id="@+id/options_activity_fragment_navigation_drawer"
-    android:name="org.oppia.app.drawer.NavigationDrawerFragment"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:layout_gravity="start"
-    app:layout="@layout/drawer_fragment"
-    tools:layout="@layout/drawer_fragment" />
-</androidx.drawerlayout.widget.DrawerLayout>
+      <include
+        android:id="@+id/options_activity_toolbar"
+        layout="@layout/toolbar" />
+
+      <FrameLayout
+        android:id="@+id/options_fragment_placeholder"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent" />
+    </LinearLayout>
+
+    <fragment
+      android:id="@+id/options_activity_fragment_navigation_drawer"
+      android:name="org.oppia.app.drawer.NavigationDrawerFragment"
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      android:layout_gravity="start"
+      app:layout="@layout/drawer_fragment"
+      tools:layout="@layout/drawer_fragment" />
+  </androidx.drawerlayout.widget.DrawerLayout>
+</layout>
+  <!-- TODO(#771): Support landscape mode. -->
```

Comment:
Remove this line
Also, EOF is missing, add one blank line at the end.

Reply:
Done